stress/common/src/main/java/alluxio/stress/worker/WorkerBenchTaskResult.java

```diff
@@ -45,13 +45,25 @@ public WorkerBenchTaskResult() {
    * @param result  the task result to merge
    */
   public void merge(WorkerBenchTaskResult result) throws Exception {
+    //When merging results within a node, we need to merge all the error information.
+    mErrors.addAll(result.mErrors);
+    aggregateByWorker(result);
+  }
+
+  /**
+   * Merges (updates) a task result with this result except the error information.
+   *
+   * @param result  the task result to merge
+   */
+  public void aggregateByWorker(WorkerBenchTaskResult result) {
+    //When merging result from different workers, we don't need to merge the error information
+    //since we will keep all the result information in a map.
```

Comment:
```suggestion
    // When merging result from different workers, we don't need to merge the error information
    // since we will keep all the result information in a map.
```